### DISCUSSION TITLE
Behaviour of Beta components doesn't follow official Beta docs?

### DISCUSSION THREAD

**Discussion by sinisarudan** [vuetifyjs/vuetify#15054] [Releases]
According to docs
https://next.vuetifyjs.com/en/api/v-select/#props-items
select's items can accept an Object:
```
const items = [
	{ text: "Florida", value: "FL" },
	{ text: "Nebraska", value: "NE" },
];
```
Yet even after update to `vuetify@v3.0.0-beta.1` it's still not supported?
Is this expected or a bug or?

I can provide more details if needed, yet dropdown texts are empty, without any thrown error.

![image](https://user-images.githubusercontent.com/1769627/166961363-14348ceb-a40e-46f6-9b3e-455747c99aa2.png)

everything else behaves normal

P.S.
Select component works correctly when strings instead of objects are used:
```
const items = ["Florida", "Nebraska"];
```
P.P.S.
` item-text` and `item-value` don't help either

**Comment by reqwire** [comment]
Same with v-chip - the text-color prop doesnt work. So many little bugs.

**Comment by sinisarudan** [comment]
Could we get some official comment on these kind of uncertainties? Thanks!

**Comment by sinisarudan** [comment]
the same issue reported here too: https://github.com/vuetifyjs/vuetify/discussions/15174

**Comment by sinisarudan** [comment]
Can anybody of the creators answer?
It's been a month, and it's a basic and very much used component
Thanks

**Reply by SeanLuis** [reply]
Have you tried being more specific in the component using item-text and item-value?

`<v-select items="items" item-text="text" item-value="value"></v-select>`

**Reply by sinisarudan** [reply]
Thanks,
but Yes of course, I've tried many documented and undocumented options :)
I've even written in my original post/question:

> P.P.S.
> item-text and item-value don't help either

**Reply by KaelWD** [reply]
#15110

**Comment by carki001** [comment]
item-text was changed to item-title
